### DISCUSSION TITLE
https://github.com/MicrosoftDocs/azure-docs/labels/triaged causes github to timeout

### DISCUSSION THREAD

**Discussion by v1934** [community/community#19269] [Other Feature Feedback, Questions, & Ideas]
I've been browsing azure docs issues and have accidentally stumbled upon a timeout with a beautiful unicorn when trying to filter the issues by the `triaged` label:

> This page is taking too long to load.

That's caused by the fact that this label is probably assigned to every single issue (over 4K), however I thought you may want to take a look into this and maybe optimize this somehow.

**Comment by 3kh0** [comment]
This happens every so often, here are a few quick fixes that sometimes help.

- Reload the page (easiest in the book)
- Reload and clear cache, in chrome do ctrl+shift+r
- Disconnect from wifi, and reconnect
- Open cmd and do `ipconfig/release` and then `ipconfig/renew`
- Turn off your computer, wait 10 seconds, and turn it back on.
- Try a different browser
- Clear cookies and site data from github.com
- Reboot your router

These are some quick and easy fixes, this issue happens from time to time again, and these fixes seem to fix it for me.

*If this helped please mark this as the answer!* ❤️
